### PR TITLE
Update page in Connect AWS Gateway “prepare-aws-api-gateway”

### PR DESCRIPTION
**Pull request by jcollins-axway**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by jcollins-axway** [comment]
@kflan-io @dfeldick @sbolosan @vivekschauhan - please review for content

**Comment by jcollins-axway** [comment]
JIRA Tickets - APIGOV-15699 APIGOV-16040 APIGOV-15737

File: content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md [hunk 1]
```diff
@@ -6,65 +6,75 @@ weight: 20
...
+## Download the CloudFormation templates
+
+```
+curl -L "https://axway.bintray.com/generic-repo/aws-agents/aws_apigw_agent_config/latest/aws_apigw_agent_config-latest.zip" -o aws_apigw_agent_config-latest.zip
+```
+
+This zip contains three files:
+
+* `cloudformation-continuous-discovery.zip`: CloudFormation templates for operating in a continuous discovery mode,
+* `cloudformation-synchronous-discovery.zip`: CloudFormation templates for operating in a synchronous discovery mode,
```
Comment (sbolosan) [inline content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md:27]: not sure what the ", " (comma) does here.  Is this a special mark down thingy?

File: content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md [hunk 2]
```diff
@@ -6,65 +6,75 @@ weight: 20
...
+{{< alert title="Note" color="primary" >}}You can either accept or update most of the template's required parameters.{{< /alert >}}
+
+## Upload all files to an S3 bucket
+
+Upload the yaml files from the `cloudformation-continuous-discovery.zip` archive and the `traceability_lambda.zip` archive to an AWS S3 bucket located on the same region the Cloud Formation template will be deployed.
+
+Note the bucket name these are uploaded to and save the URL to the `amplify-agents-deploy-all.yaml` file.
+
+Also in this bucket create a resources directory and in it create a keys directory upload the following files:
+
```
Comment (sbolosan) [inline content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md:41]: are you trying to convey to create a resources directory inside of the bucket as well as a keys directory?  or a keys directory inside of the resources directory?

**Review by kflan-io — APPROVED** [review]

File: content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md [hunk 3]
```diff
@@ -6,65 +6,75 @@ weight: 20
...
+* **ta_env_vars** which has all environment variable overrides to run the Traceability Agent
+* In the keys directory **private_key.pem** which is the private key associated with the AMPLIFY Central service account
+* In the keys directory **public_key.pem** which is the public key associated with the AMPLIFY Central service account
+
+For the values in these **\*\_env_var** files see [Deploy your agents](/docs/central/connect-aws-gateway/deploy-your-agents-1)
+
+## Create an EC2 SSH Key
+
+If you have not already, generate an SSH Key in the region that this stack will be deployed to, it is needed to assign to the EC2 instance for connecting to it
+
```
Comment (sbolosan) [inline content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md:52]: link to help create the SSH key?

File: content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md (hunk 1 above)
Comment (jcollins-axway) [reply content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md:27]: good catch, will remove

File: content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md (hunk 2 above)
Comment (jcollins-axway) [reply content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md:41]: yes I am <bucket>/resources/keys

File: content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md (hunk 3 above)
Comment (jcollins-axway) [reply content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md:52]: good call

File: content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md (hunk 2 above)
Comment (jcollins-axway) [reply content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md:41]: Within this bucket create a path of directories resources\keys:
?

Comment (sbolosan) [reply content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md:41]: yeah, i think that's better understood. To me anyways...

**Review by sbolosan — APPROVED** [review]

**Review by cbordier — APPROVED** [review]
I approve the changes.
@lbadenhop, please proceed.

**Review by lbadenhop — CHANGES_REQUESTED** [review]
Review "Create an EC2 SSH Key" changes

**Review by lbadenhop — APPROVED** [review]

**Pull request merged**